flink-kubernetes-webhook/pom.xml

```diff
@@ -127,6 +132,11 @@ under the License.
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>${surefire-plugin.version}</version>
```

Comment:
Why do we need this new plugin?

Reply:
This plugin is required for the successful execution of the test AdmissionHandlerTest.testMutateHandler. Else it will throw <500 Internal Server Error> while mutating the SessionJob as it’s using the deployment from the informerManager which uses Kubernetes Client.